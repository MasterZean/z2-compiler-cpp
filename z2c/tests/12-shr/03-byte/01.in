70u8 >> true
70u8 >> false
70u8 >> 4s8
70u8 >> 4u8
70u8 >> 4s16
70u8 >> 4u16
70u8 >> 4
70u8 >> 4u
70u8 >> 4l
70u8 >> 4ul
70u8 >> 4.0f
70u8 >> 4.0
70u8 >> '4'
70u8 >> 4p
