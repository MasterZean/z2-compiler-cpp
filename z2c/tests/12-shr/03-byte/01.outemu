Byte{4}
Byte{4}
Byte{4}
Byte{4}
Byte{4}
Byte{4}
Byte{4}
Byte{4}
Byte{4}
