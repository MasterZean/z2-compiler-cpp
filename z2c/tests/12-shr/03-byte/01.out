70u >> 4;
70u >> 4u;
70u >> 4;
70u >> 4u;
70u >> 4;
70u >> 4u;
70u >> 4ll;
70u >> 4ull;
70u >> 4u;
DummyClass(1, 6): error:
	can't apply operator '>>' on types: 
		'Byte' and 
		'Bool'
DummyClass(2, 6): error:
	can't apply operator '>>' on types: 
		'Byte' and 
		'Bool'
DummyClass(11, 6): error:
	can't apply operator '>>' on types: 
		'Byte' and 
		'Float'
DummyClass(12, 6): error:
	can't apply operator '>>' on types: 
		'Byte' and 
		'Double'
DummyClass(13, 6): error:
	can't apply operator '>>' on types: 
		'Byte' and 
		'Char'
